src/cmd.rs

```diff
@@ -19,12 +19,14 @@ pub(crate) fn to_command_response<T: Command>(
     method: MethodId,
 ) -> Result<CommandResponse<T::Response>> {
     if let Some(res) = resp.result {
-        let result = serde_json::from_value(res)?;
-        Ok(CommandResponse {
-            id: resp.id,
-            result,
-            method,
-        })
+        match serde_json::from_str(res.get()) {
```

Comment:
`from_value` takes ownership of the Value, so we need to use `RawValue` to avoid cloning it here.